src/controllers/uploadController.js

```diff
@@ -106,25 +121,82 @@ class UploadController extends PageController {
     return validationResult ? !validationResult.error : false
   }
 
-  // this function is a validation function that is called by the form wizard
-  static validateFileType ({ originalname }) {
-    const allowedFiletypes = [
-      'csv',
-      'xls',
-      'xlsx',
-      'json',
-      'geojson',
-      'gml',
-      'gpkg'
-    ]
-    // check file type
-    const fileType = originalname.split('.').pop()
-    if (!allowedFiletypes.includes(fileType)) {
+  static extensionIsValid (datafile) {
+    const allowedExtensions = ['csv', 'xls', 'xlsx', 'json', 'geojson', 'gml', 'gpkg']
+
+    const parts = datafile.originalname.split('.')
+
+    const extension = parts[parts.length - 1]
```

Comment:
This will return undefined if the filename has no extension (although that's probably ok as the check below will return false)